#! /bin/sh /usr/share/dpatch/dpatch-run
## 25_entropy.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Remove code that called netstat to gain so called entropy. It's pretty
## DP: useless on a Linux system. Thanks Wichert Akkerman. bz#51429.

@DPATCH@

--- nss/mozilla/security/nss/lib/freebl/unix_rand.c
+++ nss/mozilla/security/nss/lib/freebl/unix_rand.c
@@ -876,7 +876,11 @@
 /* Fork netstat to collect its output by default. Do not unset this unless
  * another source of entropy is available
  */
+#ifndef LINUX
 #define DO_NETSTAT 1
+#else
+#undef DO_NETSTAT 
+#endif
 
 void RNG_SystemInfoForRNG(void)
 {
